packages/file_selector/file_selector/example/lib/save_text_page.dart

```diff
@@ -5,24 +5,31 @@
 import 'dart:typed_data';
 import 'package:file_selector/file_selector.dart';
 import 'package:flutter/material.dart';
+import 'package:path_provider/path_provider.dart';
 
 /// Page for showing an example of saving with file_selector
 class SaveTextPage extends StatelessWidget {
   final TextEditingController _nameController = TextEditingController();
   final TextEditingController _contentController = TextEditingController();
 
   Future<void> _saveFile() async {
-    final String? path = await getSavePath();
-    if (path == null) {
-      // Operation was canceled by the user.
-      return;
-    }
```

Comment:
Why did you move this? Now work that is being done specifically for `saveTo` is being done further from that code, and even before we know if the save operation was cancelled or not, both of which are worse than it was before.